source/dataman/HDF5/AMHDF5File.h

```diff
@@ -14,6 +17,8 @@
 #define AMHDF5FILE_INVALID_OPEN_OPTION 666004
 #define AMHDF5FILE_FLUSH_FILE_NOT_OPEN 666005
 
+class AMHDF5Group;
```

Comment:
To compile `AMHDF5File` when it only handles references to `AMHDFGroup` required a forward declaration of the class.